package.json

```diff
@@ -85,6 +85,8 @@
   },
   "devDependencies": {
     "@colors/colors": "1.6.0",
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.10.0",
```

Comment:
These dependencies should probably be set to an exact version for consistency

Reply:
not really. These are peer dependencies and it makes sense to have them as a range to avoid unnecessary conflicts with a container package

Reply:
The `eslint-config-appium-ts` `package.json` file does indeed use ranges just like before, but this particular file is the monorepo-level `package.json`, where all other dependencies are set to exact versions.

Reply:
fixed, this was a copypaste issue